gateway/doc/p_dev_js_migrate.md

```diff
@@ -29,137 +22,26 @@ This procedure is part of [Build JavaScript WebSocket Clients](o_dev_js.md):
-
-    ``` js
-    send.onclick = function() {
-        var buf = new ByteBuffer();
-        buf.putString(message.value, Charset.UTF8);
-        buf.flip();
-        log("SENT: " + buf);
-        byteSocket.send(buf);
-    }
-    ```
-
-    Here is the equivalent function in WebSocket code from KAAZING Gateway 5.0:
-
-    ``` js
-    send.onclick = function() {
-        var buf = new Kaazing.ByteBuffer();
-        buf.putString(message.value, Kaazing.Charset.UTF8);
-        buf.flip();
-        log("SENT: " + buf);
-        websocket.send(buf);
-    }
-    ```
-
-    Note that the only different in these functions is that the ByteSocket object (`byteSocket`) is changed to a WebSocket object (`websocket`).
-
-For more information, see the ByteBuffer in the [JavaScript API](http://tech.kaazing.com/documentation/gateway/4.0/apidoc/client/javascript/gateway/index.html).
-
-Binary Support with Blob and Array Buffer
------------------------------------------
-
-In addition to including binary support with ByteBuffer, KAAZING Gateway 5.0 includes support for blob and array buffer.
-
--   **Blob** - A pointer to a single, raw binary object called a blob (binary large object). Blobs can be multimedia, images, or even executable code. Blob objects are considered safe to spool to disk because the entire data object is intact. You might use a Blob data type when you do not intend to manipulate the data, but simply write it to disk, or you might use a blob when you want to slice a file into byte ranges. The blob data type will work on any web browser that supports the [File API](http://www.w3.org/TR/FileAPI/ "File API"). For a list of web browsers that support the File API, see [caniuse.com](http://caniuse.com/#feat=fileapi).
--   **ArrayBuffer** - A fixed-length binary data buffer used to store data temporarily. You use a ArrayBufferView mask, such as Uint8Array, to view, index and manipulate the raw binary of the ArrayBuffer. There are different ArrayBufferView masks for different integer and floating point values (for example, you use a Uint16Array view to manipulate a 16-bit array of unsigned integers).
-    Use an ArrayBuffer data type instead of a Blob when you intend to work with the data, such as perform a calculation on the data. The ArrayBuffer data type is supported by web browsers that support the [Typed Array Specification](https://www.khronos.org/registry/typedarray/specs/latest/#7). For a list of web browsers that support the Typed Array Specification, see [caniuse.com](http://caniuse.com/#feat=typedarrays).
-
-Here is an example of a function using blob:
+In the KAAZING Gateway 4.x WebSocket API, the namespace object was accessed like this:
 
```

Comment:
I think we should say that in 4.x WebSocket API, all the publicly accessible javascript objects were available under window object. However, in 5.0. we introduced the namespace object Kaazing.Gateway and the javascript objects are now available under Kaazing.Gateway namespace object.